internal/sqladapter/exql/default.go

```diff
@@ -25,47 +25,50 @@ const (
 	defaultSortByColumnLayout  = `{{.Column}} {{.Order}}`
```

Comment:
It's pretty complicated to get indentation right when using strings in templates, this replaces tabs within strings to 2 spaces.